draft-irtf-panrg-path-properties.md

```diff
@@ -217,7 +219,13 @@ Finally, a NAT that actively modifies IP and TCP/UDP headers based on their cont
 Note that according to this definition, a node that modifies packets in accordance with the hosts, such as a transparent HTTP proxy, as defined in {{RFC2616}}, and a node listening and reacting to implicit or explicit signals, see {{RFC8558}}, are not considered transparent.
 
 Administrative Domain:
-: The administrative domain, e.g., the IGP area, AS, or Service provider network to which a path element belongs.
+: An individual or an organization that owns a path element (or several path elements).
+Examples of administrative domains are an IGP area, an AS, or a service provider network.
```

Comment:
Thinking about the difference between "Administrative Domain" and "Routing Domain Identifier":
Maybe the example of the administrative domain would be an identifier for the IGP area, owner of the AS, or service provider network?

Of course, we could just make this property an "Administrative domain idenfitier".

Reply:
I don't think there is a common type of identifiers for administrative domains. This is unlike the routing domain identifier property, which has a clearly defined semantics that is used by all routing domains to coordinate the routing between them.
Administrative domain seems more abstract and may not always have an identifier. However, I suppose it must have an identity to be a useful property ;D

I'm not sure how to include the word identity though...

Reply:
Do you have any suggestions how to integrate it?